notebooks/MAST/Identifying-transient-candidates/identifying_highz_candidates.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>I don't think we used lightkurve in this tutorial, so citing it isn't necessary. Since we used astropy, you can leave that part in</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>